rcl/include/rcl/publisher.h

```diff
@@ -49,6 +49,8 @@ typedef struct rcl_publisher_options_s
   rcl_allocator_t allocator;
   /// rmw specific publisher options, e.g. the rmw implementation specific payload.
   rmw_publisher_options_t rmw_publisher_options;
+  /// Disable flag to LoanedMessage, initialized via environmental variable.
```

Comment:
To make the comment clear, do you think it's necessary to add something like "True only if `ROS_DISABLE_LOANED_MESSAGES=1`"? (some cases might be used, e.g:  ROS_DISABLE_LOANED_MESSAGES=TRUE)

https://github.com/ros2/rcl/blob/c6f9fc3e78c8afdf5fd2f1ad274fb7825e05f36f/rcl/include/rcl/publisher.h#L198 should be updated

```
 * - rmw_publisher_options = rmw_get_default_publisher_options()
 * - disable_loaned_message = ?
```
Sorry, I have no good description for this.